.github/workflows/pull_request.yml

```diff
@@ -44,21 +44,34 @@ jobs:
           swift /tmp/cross-pr-checkout.swift "${{ github.repository }}" "${{ github.event.number }}"
       - name: Build
         run: swift build --configuration release
+      - name: Write check script
```

Comment:
Would it be better to re-use `github-workflows/.github/workflows/scripts/check-swift-format.sh` here and just cd to `$GITHUB_WORKSPACE/<repo>` before hand?